### PR TITLE
Betafooter

### PR DESCRIPTION
**Pull request by AlexisCatrix**
Reprise de la branche "Footer" avec conflict merging corrigés..

### CODE REVIEW

**Review by ThomasDruart — DISMISSED** [review]

**Review by gaetan-cordonnier — DISMISSED** [review]
Good

File: src/components/TeamNoctali.jsx [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+import Alexis from "../images/alexis.jpeg";
+import linkedin from "../images/linkedin.png";
+import githup from "../images/githup.png";
+import styled from "styled-components";
+
+const ContainerStyled = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  //background-color: rgb(221, 214, 214);
```
Comment (LoicBrassart) [inline src/components/TeamNoctali.jsx]: Dead code

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
One line of DC, Fix it quickly and I'll approve

**Review by ThomasDruart — APPROVED** [review]
ok, commented line deleted

**Review by gaetan-cordonnier — APPROVED** [review]

**Review by LoicBrassart — APPROVED** [review]
Most of the changes on this last commit were wild indentations
Please use the command I gave you to order this before pushing

No More Evil aside from that !

**Pull request merged**